src/tests/dbus-tests/test_80_filesystem.py

```diff
@@ -69,6 +69,12 @@ def _get_libmount_version(self):
             raise RuntimeError('Failed to determine libmount version from: %s' % out)
         return LooseVersion(m.groups()[0])
 
+    def _get_mount_options_conf_path(self):
```

Comment:
Is this needed? We are already replacing `/etc/udisks2/mount_options.conf` with the local config file in `run_tests.py` so I guess we could do the changes in the system config file here too.

Reply:
I think the problem is the recently added https://github.com/storaged-project/udisks/commit/1a9d70f0b0929c4df991b0d2b60e4236938de57a that computes config directory. When the daemon is run with `--uninstalled` as is the case for `run_tests.py` without the `--system` argument, the path is computed inside of the source tree and nothing is read from `/etc/udisks2/`. And this code snippet is about realizing from where the config file would be read.

Looks like the mentioned commit brought more changes to the tests that I originally imagined, but the paths matches and config files are read just fine.